pymove/core/dataframe.py

```diff
@@ -2395,52 +2673,67 @@ def convert_to(self, new_type):
         A subclass of MoveDataFrameAbstractModel
             The converted object.
         """
-        operation = begin_operation('convet_to')
+        operation = begin_operation("convet_to")
 
-        if (new_type == "dask"):
+        if new_type == "dask":
             _dask = DaskMoveDataFrame(
-                                      self._data,
-                                      latitude=LATITUDE,
-                                      longitude=LONGITUDE,
-                                      datetime=DATETIME,
-                                      traj_id=TRAJ_ID,
-                                      n_partitions=1)
+                self._data,
+                latitude=LATITUDE,
+                longitude=LONGITUDE,
+                datetime=DATETIME,
+                traj_id=TRAJ_ID,
+                n_partitions=1,
+            )
             self.last_operation = end_operation(operation)
             return _dask
-        elif (new_type == "pandas"):
+        elif new_type == "pandas":
             self.last_operation = end_operation(operation)
             return self
 
     def get_type(self):
-        """Returns the type_ of the object.
+        """
+        Returns the type_ of the object.
 
         Returns
         -------
         A string representing the type_ of the object.
         """
-        operation = begin_operation('get_type')
+        operation = begin_operation("get_type")
         type_ = self._type
         self.last_operation = end_operation(operation)
         return type_
 
 
 class DaskMoveDataFrame(DataFrame, MoveDataFrameAbstractModel):
-    def __init__(self, data, latitude=LATITUDE, longitude=LONGITUDE, datetime=DATETIME, traj_id=TRAJ_ID,
-                 n_partitions=1):
+    def __init__(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__ (D107)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5421338)